app/root.tsx

```diff
@@ -60,6 +62,12 @@ export function links() {
 export async function loader({context}: LoaderArgs) {
   const {storefront, session, cart} = context;
   const customerAccessToken = await session.get('customerAccessToken');
+
+  // Set customerAccessToken as a Sentry tag
+  Sentry.configureScope(scope => {
+    scope.setTag('customerAccessToken', customerAccessToken?.accessToken);
```

Comment:
Let's not add this as a tag, from what I see in the docs it works like a session token, so I don't think it's huge value to users.